File: Cafebook/CafebookModel/Model/ModelWeb/KhachHang/LichSuDatBanDto.cs
using System;
using System.Collections.Generic;

namespace CafebookModel.Model.ModelWeb.KhachHang
{
    public class LichSuDatBanDto
    {
        public int IdPhieuDatBan { get; set; }
        public string TenBan { get; set; } = string.Empty;
        public DateTime ThoiGianDat { get; set; }
        public int SoLuongKhach { get; set; }
        public string TrangThai { get; set; } = string.Empty;
        public string? GhiChu { get; set; }
    }

    public class PagedLichSuResponseDto
    {
        public List<LichSuDatBanDto> Items { get; set; } = new List<LichSuDatBanDto>();
        public int TotalCount { get; set; }
        public int TotalPages { get; set; }
        public int CurrentPage { get; set; }
    }
}

File: Cafebook/ADRCafebookApi/app/src/main/java/com/example/cafebook/models/LichSuDatBanDto.java
package com.example.cafebook.models;

import com.google.gson.annotations.SerializedName;
import java.util.List;

public class LichSuDatBanDto {

    public static class Response {
        @SerializedName("items") public List<Item> items;
        @SerializedName("totalCount") public int totalCount;
        @SerializedName("totalPages") public int totalPages;
        @SerializedName("currentPage") public int currentPage;
    }

    public static class Item {
        @SerializedName("idPhieuDatBan") public int idPhieuDatBan;
        @SerializedName("tenBan") public String tenBan;
        @SerializedName("thoiGianDat") public String thoiGianDat; // ISO 8601
        @SerializedName("soLuongKhach") public int soLuongKhach;
        @SerializedName("trangThai") public String trangThai;
        @SerializedName("ghiChu") public String ghiChu;
    }
}
